src/main/java/com/checkconsulting/proepargne/repository/TransactionRepository.java

```diff
@@ -16,4 +17,7 @@ public interface TransactionRepository extends JpaRepository<Transaction, Long>
     @Query("SELECT t FROM Transaction t WHERE t.account IN :accounts AND (LOWER(t.type) LIKE LOWER(CONCAT('%', :filter, '%')) or :filter is null or :filter='')  ")
     Page<Transaction> findAllByAccountIn(List<Account> accounts, Pageable pageable, String filter);
 
+    @Query("SELECT t FROM Transaction t WHERE (:planType is null or t.planType = :planType or :planType = '')")
```

Comment:
Do you mean either this `planType` or null ?

Reply:
null or empty or = planType there's anythore way to do it ?

Reply:
Even tho planType will never be null

Reply:
![d962ff20-a94f-4c6d-bb23-196e41f9a6b5](https://github.com/mchekini-check-consulting/pro-epargne-api/assets/61586802/ef8edf73-65f8-45b9-b1e1-6f39b0c58f18)

Reply:
i think maybe there's a way i'm trying to find something